### ISSUE TITLE
[Broken Website] Dark Reader doesn't work on google docs and google sheets in Firefox

### ISSUE THREAD

**Issue by Prakashh21** [darkreader/darkreader#9061]
# Broken Website Report

## Website Address

https://docs.google.com/
https://docs.google.com/spreadsheets/

## Steps To Reproduce

- Go to `https://docs.google.com/`
- or go to `https://docs.google.com/spreadsheets/`
- enable dark reader using ctrl+alt+D.

## Problematic Observation

dark reader doesnot work on https://docs.google.com/ and https://docs.google.com/spreadsheets/

## System Information

- OS:  Arch Linux kernel -- 5.18.2-arch1-1
- Browser:  Firefox 101.0 (64-bit)
- Dark Reader Version:    4.9.51

## Additional Context

This issue is not present on google chrome. dark reader works completely fine in google docs and google sheets while using chrome.

**Comment by Gusted** [comment]
Hi @Prakashh21!

Do you mean that dark reader isn't modifying those sites? You're not quite clear on the problematic observation for this issue.

**Comment by Prakashh21** [comment]
@Gusted  yes it is not able to modify those sites... another important info that I missed was.. I am using this firefox css theme.. https://github.com/mut-ex/minimal-functional-fox 

and had modified these settings in firefox to true in order to use that theme.

```
toolkit.legacyUserProfileCustomizations.stylesheets
layers.acceleration.force-enabled
gfx.webrender.all
gfx.webrender.enabled
layout.css.backdrop-filter.enabled
svg.context-properties.content.enabled
```

**Comment by Prakashh21** [comment]
@Gusted  any idea why this is happening

**Comment by Gusted** [comment]
> @Gusted yes it is not able to modify those sites... another important info that I missed was.. I am using this firefox css theme.. https://github.com/mut-ex/minimal-functional-fox
> 
> and had modified these settings in firefox to true in order to use that theme.
> 
> ```
> toolkit.legacyUserProfileCustomizations.stylesheets
> layers.acceleration.force-enabled
> gfx.webrender.all
> gfx.webrender.enabled
> layout.css.backdrop-filter.enabled
> svg.context-properties.content.enabled
> ```

Please try to turn of those otpions and check if it works then.

**Comment by Prakashh21** [comment]
Hey @Gusted It was resolved in the Firefox version --  Firefox 101.0.1 , I am closing this issue..!
